Description: Avoid a linker bug where symbols are not found.
 > /usr/bin/ld.bfd.real: out/Release/obj.target/net/libnet.so: undefined reference to symbol 'SSL_ForceHandshake'
 > out/Release/obj.target/net/third_party/nss/libcrssl.so: error adding symbols: DSO missing from command line

Index: dev.utopic/build/common.gypi
===================================================================
--- dev.utopic.orig/build/common.gypi	2014-09-08 19:47:10.823758349 -0400
+++ dev.utopic/build/common.gypi	2014-09-08 19:47:10.820758349 -0400
@@ -3570,7 +3570,7 @@
                   '-fomit-frame-pointer',
                 ],
               }],
-              ['OS=="linux" and target_arch=="ia32"', {
+              ['OS=="linux"', {
                 'ldflags': [
                   '-Wl,--no-as-needed',
                 ],
@@ -3617,7 +3617,6 @@
               # Specifically tell the linker to perform optimizations.
               # See http://lwn.net/Articles/192624/ .
               '-Wl,-O1',
-              '-Wl,--as-needed',
             ],
             'conditions' : [
               ['no_gc_sections==0', {
@@ -3669,6 +3668,11 @@
               }, {
                 'cflags': ['-fno-unwind-tables', '-fno-asynchronous-unwind-tables'],
               }],
+              ['OS=="linux"', {
+                'ldflags': [
+                  '-Wl,--no-as-needed',
+                ],
+              }],
             ],
           },
         },
